### ISSUE TITLE
Histogram Chart?

### ISSUE THREAD

**Issue by pietz** [frappe/charts#408]
I would like to visualize a histogram of values similar to what many libraries like [plotly](https://plotly.com/python/histograms/) offer. Of course it would be great for Frappe to do this out of the box, but I don't mind creating the bins myself. However, even then I'm facing a problem:

In a bar chart each bar is always associated with 1 label, while I'd like to use a bar to represent a range of values. As a workaround it would be nice if I could define the labels separate from the data. For example, if the labels are exactly 1 element longer than the values, it could be interpreted as the bar edges.

In summary:

1. I think a histogram plot would be great addition to Frappe Charts
2. At least it would be nice to create a workaround for which we would need to decouple the labels from the values.

**Comment by vanshgupta11** [comment]
I could find the best solution for it . So assign this task to me .
